lib/output.js

```diff
@@ -597,6 +598,12 @@ function OutputStream(options) {
             return true;
         }
 
+        if (output.option('webkit')
+            && output.parent(0) instanceof AST_PropAccess
+            && output.parent(1) instanceof AST_Assign) {
```

Comment:
> I noticed a few single quotes in recent commits for string literals.

That was a mess-up on my part - I was too busy getting Node.js REPL to list the candidates correctly that I forgot to replace all the single-quotes to double-quotes in the end. Sorry about that.